Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -22,6 +22,14 @@ public struct WorldScaleGeoTrackingSceneView: View {
     @State private var arViewProxy = ARSwiftUIViewProxy()
     /// The camera controller that will be set on the scene view.
     @State private var cameraController: TransformationMatrixCameraController
+    /// The camera controller heading.
+    @State var heading: Double = 0
+    /// The camera controller elevation.
+    @State var elevation: Double = 0
+    /// The calibrated elevation delta.
+    @State private var elevationDelta: Double = 0
```

Comment:
I wouldn't think you would need both the elevation and the elevation delta. I think you should be able to calculate the delta from the current elevation: cameraController.originCamera.location.z.

Reply:
The elevation delta can be calculated, I will remove `elevationDelta`.